### DISCUSSION TITLE
Garageband＋ChatGPT 初體驗

### DISCUSSION THREAD

**Discussion by JohnnyXiao1024** [1101-GS4719-C/Python-Community-GS4719-C#1259] [General]
剛好這禮拜要考的科目都沒了
所以就跑去玩了之前一直想碰碰看的Garageband
雖然自己根本是顆音樂麻瓜QQ

先花個十分鐘了解一下和弦原理
再順手跟chatGPT要了一組和弦
聽一聽覺得無趣 就發現網路上說要加琶音
然後亂塞一些自己喜歡的東東
<img width="822" alt="截圖 2023-04-20 上午2 20 52" src="https://user-images.githubusercontent.com/125842539/233167736-17c9e0bf-4dd1-4f3e-beef-684fc2946031.png">

不怕耳朵爛掉的可以聽：
[https://www.youtube.com/watch?v=9FiIExtRIVs......](https://www.youtube.com/watch?v=9FiIExtRIVs&ab_channel=little_johnny&fbclid=IwAR0UsYbPjgRhUo6AWXucbAwgzK6SocQu9zf1Nfwc_8LzGKL2la__6TWf8C4)

**Comment by JohnnyXiao1024** [comment]
我是參考[這支影片](https://www.youtube.com/watch?v=-Wuj5CrmGfw&ab_channel=PAPAYA%E9%9B%BB%E8%85%A6%E6%95%99%E5%AE%A4)邊學邊做的。

這個頻道還有許多實用的教學，包含PPT、excel或各種ai工具
影片內容簡潔好懂，沒有任何一句廢話，非常值得推薦給大家～

**Comment by curtis1017** [comment]
我覺得這也太酷了吧，Chatgpt可以用在音樂上，我相信未來可以應用在更多方面上，看了你的文章後，好奇心驅使我也去嘗試看看。

**Comment by lyien1021** [comment]
好酷！之前我也有試玩GarageBand，不過沒想到可以跟ChatGPT結合使用，之後有空也要來試試看！

**Comment by DuxtinTai** [comment]
原來在音樂上也可以用得上阿，會不會很快chatgpt就可以搭配語音了，真讓人期待!
btw，我之前有嘗試過用於旅行規劃，對於大方向設定還有景點找尋效果也很不錯!
